samples/dnn/yolo_object_detection.cpp

```diff
@@ -1,3 +1,23 @@
+//! Brief Sample of using OpenCV dnn module in real time with device capture, video and image.
+//! Questions and suggestions email to: Alessandro de Oliveira Faria cabelo@opensuse.org or blog http://assuntonerd.com.br
+/*
+ * VIDEO DEMO:
+ * https://www.youtube.com/watch?v=NHtRlndE2cg
+ *
+ * COMPILE:
+ * g++ `pkg-config --cflags opencv` `pkg-config --libs opencv` yolo_object_detection.cpp -o yolo_object_detection
+ *
+ * RUN in webcam:
```

Comment:
I think it's better to add mode details to command line arguments instead. In example `"{ class_names    |       | File with class names, [PATH-TO-DARKNET]/data/coco.names }";`

Reply:
ok

Reply:
Done